docs/build.md

```diff
@@ -280,19 +280,27 @@ cmake -B build -DGGML_VULKAN=ON
 cmake --build build --config Release
 ```
 
-**With docker**:
+**With containers**:
 
 You don't need to install Vulkan SDK. It will be installed inside the container.
 
 ```sh
 # Build the image
 docker build -t llama-cpp-vulkan --target light -f .devops/vulkan.Dockerfile .
 
+or
+
+podman build -t llama-cpp-vulkan --target light -f .devops/vulkan.Dockerfile .
+
 # Then, use it:
 docker run -it --rm -v "$(pwd):/app:Z" --device /dev/dri/renderD128:/dev/dri/renderD128 --device /dev/dri/card1:/dev/dri/card1 llama-cpp-vulkan -m "/app/models/YOUR_MODEL_FILE" -p "Building a website can be done in 10 simple steps:" -n 400 -e -ngl 33
+
+or
```

Comment:
same here, or probably split it into 2 different code blocks to avoid bad copy/paste when user don't know much about it